lib/general.sh

```diff
@@ -272,26 +272,35 @@ improved_git()
 
 }
 
-clean_up_repo ()
+clean_up_git ()
 {
 	local target_dir=$1
 
 	# Files that are not tracked by git and were added
 	# when the patch was applied must be removed.
-	improved_git -C $target_dir clean -qdf
+	git -C $target_dir clean -qdf
```

Comment:
why not "improved_git" ?

Reply:
I realized that this replacement of a simple git command is necessary when git accesses a remote repository. What is the need to use it in all cases?

Reply:
https://github.com/armbian/build/blob/master/lib/general.sh#L254

Reply:
I probably didn't understand something in this line. I'll fix it now.

Reply:
There are more git calls in this file. Change also those.

Reply:
> https://github.com/armbian/build/blob/master/lib/general.sh#L254

I understood this line literally like this:
If the git team needs to call a remote server, use this function. It will repeat the command if the server does not respond for a long time.

Reply:
I tested it with a remote server and it really works well. But I didn't see the point of using it for local actions.

Reply:
Fix those that are calling remote git. It should be done quickly. Github acts strangly from time to time and this workaround is usually enough to not break build process.